src/pkg/mcp/prompts/awsBYOC.go

```diff
@@ -30,7 +28,12 @@ func setupAWSBYOPrompt(s *server.MCPServer, cluster string, providerId *client.P
 		),
 	)
 
-	s.AddPrompt(awsBYOCPrompt, func(ctx context.Context, req mcp.GetPromptRequest) (*mcp.GetPromptResult, error) {
+	s.AddPrompt(awsBYOCPrompt, AWSBYOCPromptHandler(cluster, providerId))
+}
+
+// AWSBYOCPromptHandler is extracted for testability
+func AWSBYOCPromptHandler(cluster string, providerId *client.ProviderID) func(ctx context.Context, req mcp.GetPromptRequest) (*mcp.GetPromptResult, error) {
```

Comment:
lower case, ie. not exported? Also, `…AwsByoc…` is more readable than `AWSBYOC…`